### ISSUE TITLE
The Audio Volume seems way higher here compared to other games

### ISSUE THREAD

**Issue by Gagert** [Rubberduckycooly/Sonic-1-2-2013-Decompilation#147]
I've just compared the audio volume to this decompilation to other games on my computer, and at least in Windows 10 the audio volume seems fairly high here. This is a very minor issue in my opinion, but this maybe worth noting nonetheless.

The same goes for the Sonic CD decompilation as well (I haven't created a thread in the Issue tracker on that repository regarding this though).

**Comment by ffoxD** [comment]
You can change audio and music volume separately in settings.ini.

`[Audio]
BGMVolume=1.000000
SFXVolume=1.000000`

**Comment by Gagert** [comment]
Late reply from my part. My bad.

I have now played some with my volume sliders on the newest release of this decompilation of Sonic 1 & 2. I've compared various audio volume values to games which I already have in my Steam library.  I've tested a few games there with it's volume settings set to the highest value (often referred to 10 or 100), with some of my testing candidates being Batman Arkham City, A Hat in Time and last but not least Sonic Mania itself.

Of all those games I believe Sonic Mania is the best candidate to compare audio volumes on full volume (1, 10, 100 or whatever the selected game names its max volume). Comparing audio volumes with Green Hill Zone of Sonic 1 decompiled and Sonic M;ania, with Sonic Mania on max volume, I personally found that audio volumes of 0.3 on Sonic 1 decompiled matched best the max volume of Sonic Mania. Thereby my personally preferred settings for audio volume of the Sonoc decompilation projects here are as follows:

[Audio]
BGMVolume=0.3
SFXVolume=0.3

0.3 is way below the standard value of 1.0 that's default in the Sonic decompilation projects audio volumes here.

Could lowering the audio volume of the default value of 1.0 to something more like the volume at 0.3 be more ideal, at least of the Windows releases of this decompilation?
